src/onnxruntime.cc

```diff
@@ -700,18 +702,16 @@ ModelState::AutoCompleteConfig()
       std::string kind_str;
       RETURN_IF_ERROR(instance_group_kind.AsString(&kind_str));
 
-      if (kind_str == "KIND_GPU" || kind_str == "KIND_AUTO") {
+      if (kind_str == "KIND_GPU") {
         found_gpu_instance = true;
```

Comment:
Does it make sense to just set the kind here and break rather than toggling the bool?
```cpp
    if (kind_str == "KIND_GPU") {
      kind = TRITONSERVER_INSTANCEGROUPKIND_GPU;
      break
    }
```

Reply:
good find!

Reply:
Fixed

Reply:
I believe the ask here was to remove `found_gpu_instance` bool variable and directly set the kind to GPU here.

Reply:
Forgot to push the change after testing. updated